crates/platform-tags/src/tags.rs

```diff
@@ -753,6 +750,158 @@ mod tests {
         ]
         "###
         );
+
+        let tags = compatible_tags(&Platform::new(
+            Os::Macos {
+                major: 14,
+                minor: 0,
+            },
+            Arch::X86_64,
+        ))
+        .unwrap();
```

Comment:
I added some new coverage for x86_64 because it has a separate branch of logic but these were correct without changes.